backend/src/models/v2/User.ts

```diff
@@ -0,0 +1,32 @@
+import { Document, model, Schema } from 'mongoose'
+
+// This interface stores information about the properties on the base object.
+// It should be used for plain object representations, e.g. for sending to the
+// client.
+export interface UserInterface {
+  // Do not store user role information on this object.  This information
+  // should be stored in an external corporate store.
+  dn: string
```

Comment:
Are we expecting every auth provider to include a 'dn' in users' information?

Reply:
We can't use the word `id` as it's restricted by Mongoose.  Thus, I figure `dn` is a common enough phrase to get the point across of having a single globally unique identifier per person?  I'm happy to change it if a better name is around that doesn't relate to LDAP servers.